app/views/superpowers/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for @superpower do |f| %>
+
+<% if @superpower.errors.size > 0 %>
+  <ul>
+    <% @superpower.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+  </ul>
+<% end %>
+
+<div>
+  <%= f.label :name, 'Superpower'%>
+  <%= f.text_area :name %>
+</div>
+
+<div>
+  <%= f.label :description, 'Description' %>
+  <%= f.text_area :description %>
+</div>
+
+  <%= f.submit 'Add Superpower' %>
```

Comment:
consider tabbing this in or putting it in a `<div>` because the button is an inline element